op-chain-ops/interopgen/deploy.go

```diff
@@ -308,9 +307,10 @@ func GenesisL2(l2Host *script.Host, cfg *L2Config, deployment *L2Deployment) err
 		L1FeeVaultWithdrawalNetwork:              big.NewInt(int64(cfg.L1FeeVaultWithdrawalNetwork.ToUint8())),
 		GovernanceTokenOwner:                     cfg.GovernanceTokenOwner,
 		Fork:                                     big.NewInt(cfg.SolidityForkNumber(1)),
-		UseInterop:                               cfg.UseInterop,
-		EnableGovernance:                         cfg.EnableGovernance,
-		FundDevAccounts:                          cfg.FundDevAccounts,
+		// Only include interop predeploys if it is activating at genesis
+		UseInterop:       cfg.L2GenesisInteropTimeOffset != nil && *cfg.L2GenesisIsthmusTimeOffset == 0,
```

Comment:
It would be a good follow up to remove the `useInterop` from the solidity side too

Reply:
We will always need UseInterop on the solidity side to control whether or not it should deploy the interop redeploys in genesis.  In fact it will wind up being two flags to control if the CrossL2InboxProxy should be initialised or not (ie are you in a dep set > 2).  These are just akin to the `EnableGovernance` and `FundDevAccounts` flags that control what gets deployed.